.travis.yml

```diff
@@ -28,7 +28,7 @@ env:
         - secure: "GROiBt5zib8ZYg2CqLJMdfIRgFxqYX7rPbzedtHU7L+YTEsacj/HbbzEwWVysMwDoMs5MWGlOoK6c0yCZEVrD5WisE2PN3xLTNzJkNSqPyHH4SLY3+LgVo6416sP3kNh3xftmeyY5pWSX3WhFLaI+BdPZNSYSatfcQcqsJRn3KHeWaDwUFjYxhJ4jVsZB+ELFQAlLU3gpC5yCKXOaB9BMwLSJai8a9BGpfSFjkOzr9RgeAdaZutHYEmmHpv91sjtS/GC9Jw6991L/58KWHqe+NxURg3HIziYWoZxYiRFojN4A9Bn+n4FbzWOA/ELOG+D95P9JeYpq+IS3g0j2a14J9UQK7sz6Rqa/niO7pkYBPn1Wp+rOyaw1Ietmcnu3OzK6cD8BBQH0D/4IRxNDgBKf3oUyt2fVAqgpm6LQ//yUaD7alvj7Z0oaGC1MQwooiF/3Bz6fcxE+dmIECVsDFrJ2+sBad3gm2fuHP0Ng33ZyoIn0ehu2Byrc0FB6/lXYUmIDxEhpQQDL0Qs6IBQV5uXolPrDZAhOlhzLg4OFvDABkUFehwroHJSRRjtSU8Ok0GAiiaInteMatTvmoyPYJ8TM+fElOqXnZU07UlZ4cqCM1v2MyZE4JOW0gR6lBm7XjbI1jbo2Etf+CQMYCf6f2HcnTBAHhvBIxJWaJRAWYVnq0c="
 
         # Add the TRAVIS_TOKEN environment variable for the "conda-forge-manager" Travis user.
-        - secure: "R3jsXK656gTxvty2mP04P17vQGFKqi5NsvdFriA1gcP0koj81PqNyBaYFfNSK7PECTLSJtJsZvfyyLN11x8mHn0lPNkPdpKHefg4O1kJUd+mDuKPXg2HRguWFR/79iHgo6p/pxD6nANaXZui8MeLX6O5FziSFlPcLE3c+c1PFlhJytA7pm7g1096wKFU6uONPV10u6NFUupNNzLx6sUbGnVx9TzLQ/JAv9oUeERfyAIeGaU6gSJAmF7+DruqmjzMGz5DHp/3Wizxx110f9m2Tp99qIMhXQ2Hq9aVxh9x2I7xO3feITCK0+2hI5dDbalGht/fumzI4aQpRSWVHmVEOnf9KfuunF1GgqwC2DVZKpIexYz7Rey3/8uyKB7tZ5Ab83gEbNvPbN/ijPOY/qWE1CEdDjuYoH12KdEAHFnsDFGyUpBTVjWKpmmPP0J48JDpnzt9znYVW1G9qKmaoPGEFRSdeYEy/lyU4adTnquQznlMZLj+QUFYtAKFzKfS1yKLYtoAFB03nON3T6zTiNEXB2Yv3/8iv1xv4QpfPyJgDr8V4KbvUM7m6RyPqN/4IPIHgYrCHK7T3VX8nhbQorEko6Dq2vO/pTSICKhFHNL5BKMjMhHPvwmhcna4XtJaWSEFFMifk14ZvYt5SPHW5jaoA4Cav9Xi7EegYcT77vM73nc="
+        # - secure: "R3jsXK656gTxvty2mP04P17vQGFKqi5NsvdFriA1gcP0koj81PqNyBaYFfNSK7PECTLSJtJsZvfyyLN11x8mHn0lPNkPdpKHefg4O1kJUd+mDuKPXg2HRguWFR/79iHgo6p/pxD6nANaXZui8MeLX6O5FziSFlPcLE3c+c1PFlhJytA7pm7g1096wKFU6uONPV10u6NFUupNNzLx6sUbGnVx9TzLQ/JAv9oUeERfyAIeGaU6gSJAmF7+DruqmjzMGz5DHp/3Wizxx110f9m2Tp99qIMhXQ2Hq9aVxh9x2I7xO3feITCK0+2hI5dDbalGht/fumzI4aQpRSWVHmVEOnf9KfuunF1GgqwC2DVZKpIexYz7Rey3/8uyKB7tZ5Ab83gEbNvPbN/ijPOY/qWE1CEdDjuYoH12KdEAHFnsDFGyUpBTVjWKpmmPP0J48JDpnzt9znYVW1G9qKmaoPGEFRSdeYEy/lyU4adTnquQznlMZLj+QUFYtAKFzKfS1yKLYtoAFB03nON3T6zTiNEXB2Yv3/8iv1xv4QpfPyJgDr8V4KbvUM7m6RyPqN/4IPIHgYrCHK7T3VX8nhbQorEko6Dq2vO/pTSICKhFHNL5BKMjMhHPvwmhcna4XtJaWSEFFMifk14ZvYt5SPHW5jaoA4Cav9Xi7EegYcT77vM73nc="
```

Comment:
You can't drop the key like this. You have to remove the line where `.conda-smithy/travis.token` is created